Tasks/ArchiveFilesV2/archivefiles.ts

```diff
@@ -120,6 +120,11 @@ function sevenZipArchive(archive: string, compression: string, files: string[])
         sevenZip.arg('-mx=' + mapSevenZipCompressionLevel(sevenZipCompression));
     }
 
+    const decoderCompatibility = tl.getInput('decoderCompatibility', false);
+    if (win && decoderCompatibility && archiveType == '7z') {
```

Comment:
By the PR description, should it be `!win` as it's only for linux?

Reply:
linux uses old version to archive and extract, but windows uses new version decoder for both.
if we archive on windows (new) and extract on linux (old version) it fails. 
thus, we provide new param on windows only (new version 7z), old versions (linux) doesn't have this argument